modules/circleci/settings.go

```diff
@@ -15,15 +15,17 @@ const (
 type Settings struct {
 	common *cfg.Common
 
-	apiKey string `help:"Your CircleCI API token."`
+	apiKey      string `help:"Your CircleCI API token."`
+	buildNumber int    `help:"The number of build, 10 by default"`
```

Comment:
I would suggest `numberOfBuilds` instead. `buildNumber` could be misconstrued as a specific build number (ie. 453) rather than a quantity.

Reply:
Agreed. Adressed this in 5b17ddf, thanks for the review. 
Not sure why the linting fails in https://github.com/wtfutil/wtf/pull/1025/checks?check_run_id=1421195730 though.